ecl/eclagent/eclagent.cpp

```diff
@@ -575,7 +575,8 @@ EclAgent::EclAgent(IConstWorkUnit *wu, const char *_wuid, bool _checkVersion, bo
             abortmonitor->setGuillotineCost(money2cost_type(guillotineCost));
         }
     }
-
+    if (isContainerized())
+        configurePreferredPlanes();
```

Comment:
question: Should this be bare-metal as well?  Any reason to restrict it?

Reply:
other than there isn't a way to configure in bare-metal at the moment, no, the code will 'work'.
I'll remove the isContainerized()